### PR TITLE
Fix cgroup stat collector and build tags

### PR DESCRIPTION
**Pull request by marceloamaral**
There is a bug in the current code that collects cgroup metrics.
We are calling the stat function only once, so that we do not update the cgroup metrics and are only using old values.

This PR fix this.

### CODE REVIEW

**Comment by jichenjc** [comment]
not sure why I can't trigger merge.. @marceloamaral maybe you can merge by yourself

**Comment by marceloamaral** [comment]
@jichenjc 
humm, do you have the right to approve a PR? 
If yes, I need an approval before merging this PR.

**Review by jichenjc — APPROVED** [review]

**Comment by jichenjc** [comment]
ok, I thought we use /approve /lgtm to auto merge :) 
but seems we don't yet , but anyway, glad it merged

**Review by SamYuan1990 — COMMENTED** [review]
LGTM

**Pull request merged**
